packages/ui/src/composites/wui-list-button/styles.ts

```diff
@@ -0,0 +1,18 @@
+import { css } from 'lit'
+
+export default css`
+  button {
+    column-gap: var(--wui-spacing-s);
+    padding: 16.5px var(--wui-spacing-l) 16.5px var(--wui-spacing-xs);
+    width: 100%;
+    background-color: var(--wui-gray-glass-002);
+    border-radius: var(--wui-border-radius-xs);
+    color: var(--wui-color-fg-100);
+    justify-content: center;
+    align-items: center;
+  }
+  button:disabled {
```

Comment:
Let's separate styles with new line

```suggestion

  button:disabled {
```